lib/utils/logDeprecation.js

```diff
@@ -97,7 +78,7 @@ module.exports = (code, message, { serviceConfig } = {}) => {
           `REJECTED_DEPRECATION_${code}`
         );
       case 'warn':
-        writeDeprecation(code, message);
+        // TODO: same handling as warn?
```

Comment:
This seems to me like a small ommision when changing how deprecations are logged - at the moment, we don't have support for `warn` mode - is that intentional? Should we treat `warn` and `warn:summary` the same way now?

Reply:
I think I've assumed that both `warn` and `warn:summary` will fallback to `sls doctor`, but indeed it'll be better to have `warn` show logs explicitly (I can imagine some users, not liking need of next step to see the deprecations, and this is the mode for that)

Reply:
So, would you suggest that we log in a similar way as we did previously in `warn` mode instead of going the `sls doctor` route?

Reply:
Yes, I believe it's good to keep such option, still I think this update doesn't belong there, but to `master` branch (for completeness)